index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    <link rel="stylesheet" href="styles.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@300&family=Poppins:wght@600&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+
+    <title>project 2- product card Organic Fruit Juice</title>
+  </head>
+
+  <body>
+    <div class="container">
+      <img
+        src="https://img.freepik.com/free-psd/colorful-delicious-smoothies-mock-up_23-2148237106.jpg?size=626&ext=jpg"
+        alt="organic juices mockup image"
+        width="350"
+      />
+      <h2>
+        Organic Fruit Juice
+      </h2>
+      <p>
+        Made with 100% organic fruits, lots of vitamin c and fiber. Comes with
+        three sets- banana, orange and plum juice.
+      </p>
+      <h3>
+        $9.99
+      </h3>
+      <button class="share">
+        <i
+          class="fa fa-twitter"
+          style="font-size:28px"
+          onclick="window.location.href='https://twitter.com/intent/tweet?text=https://ymoon-project2.glitch.me/';"
+        ></i>
+      </button>
+      <button class="like">
+        <i class="material-icons">favorite</i>
+      </button>
+      <button class="add_cart">
+        <i class="material-icons">shopping_cart</i>
+      </button>
```

Comment:
Each of these `button` elements needs a `type` attribute, like so:

`<button class="share" type="button">`

💡 This is something we discussed in class _after_ project 2, so no worries!